app/src/main/java/emu/skyline/utils/PreferenceSettings.kt

```diff
@@ -25,6 +25,7 @@ class PreferenceSettings @Inject constructor(@ApplicationContext private val con
     // System
     var isDocked by sharedPreferences(context, true)
     var usernameValue by sharedPreferences(context, context.getString(R.string.username_default))
+    var profilePictureValue by sharedPreferences(context, "")
```

Comment:
Name this preference `profilePicturePath` instead of value. Reflect this change on the C++ side as well.